addons/web/controllers/binary.py

```diff
@@ -179,9 +179,10 @@ def content_assets(self, filename=None, unique=ANY_UNIQUE, nocache=False, assets
     def content_image(self, xmlid=None, model='ir.attachment', id=None, field='raw',
                       filename_field='name', filename=None, mimetype=None, unique=False,
                       download=False, width=0, height=0, crop=False, access_token=None,
-                      nocache=False):
+                      nocache=False, **kwargs):
         try:
-            record = request.env['ir.binary']._find_record(xmlid, model, id and int(id), access_token, field=field)
+            record = request.env['ir.binary']._find_record(xmlid, model, id and int(id),
+                                                           access_token, field=field, **kwargs)
```

Comment:
As long as they are used in a safe manner this is not an issue, but we can always filter the kwargs we let pass-through if we want more guarantees, this can't hurt.